clients/vscode-hlasmplugin/.vscode/launch.json

```diff
@@ -1,30 +1,36 @@
 // A launch configuration that compiles the extension and then opens it inside a new window
 {
-    "version": "0.1.0",
+    "version": "0.2.0",
     "configurations": [
         {
             "name": "Extension Tests",
             "type": "extensionHost",
             "request": "launch",
             "runtimeExecutable": "${execPath}",
             "args": [
-              "${workspaceFolder}/lib/test/workspace/",
-              "--disable-extensions",
-              "--extensionDevelopmentPath=${workspaceFolder}",
-              "--extensionTestsPath=${workspaceFolder}/lib/test/suite/index"
+                "${workspaceFolder}/lib/test/workspace/",
+                "--disable-extensions",
+                "--extensionDevelopmentPath=${workspaceFolder}",
+                "--extensionTestsPath=${workspaceFolder}/lib/test/suite/index"
+            ],
+            "sourceMaps": true,
+            "outFiles": [
+                "${workspaceFolder}/lib/**/*.js"
             ],
-            "outFiles": ["${workspaceFolder}/lib/test/**/*.js"],
             "preLaunchTask": "prepareTest"
         },
         {
             "name": "Launch Extension",
             "type": "extensionHost",
             "request": "launch",
             "runtimeExecutable": "${execPath}",
-            "args": ["--extensionDevelopmentPath=${workspaceRoot}" ],
-            "stopOnEntry": false,
+            "args": [
+                "--extensionDevelopmentPath=${workspaceFolder}"
```

Comment:
The stopOnEntry has not effect here?

Reply:
Nope, not in this case. It even says the following:
![image](https://user-images.githubusercontent.com/99467904/224961684-feb628f4-dd6e-4602-99a4-40702f2466ee.png)